### PR TITLE
:lipstick:

### PR DESCRIPTION
**Pull request by SamVerschueren**
couldn't resist...

### CODE REVIEW

**Comment by sindresorhus** [comment]
:lipstick: is more fitting ;)

**Comment by SamVerschueren** [comment]
Good point. From now on, 💄 it is ;)

File: readme.md [hunk 1]
```diff
@@ -13,6 +13,8 @@ $ npm install --save is-svg
 ## Usage
 
 ```js
```
Comment (sindresorhus) [inline readme.md]: Can you remove the `sh` on line 8 and add `http` => `https` on the last line?

**Comment by SamVerschueren** [comment]
Done. Added `https` in package.json as well.

File: package.json [hunk 2]
```diff
@@ -7,13 +7,13 @@
   "author": {
     "name": "Sindre Sorhus",
     "email": "sindresorhus@gmail.com",
-    "url": "http://sindresorhus.com"
+    "url": "https://sindresorhus.com"
```
Comment (sindresorhus) [inline package.json]: `https://sindresorhus.com` => `sindresorhus.com`

I do that for simplicity.

**Comment by SamVerschueren** [comment]
Et voila

**Comment by sindresorhus** [comment]
Awesome :)

**Comment by SamVerschueren** [comment]
LOL :D

screen shot 2016-04-21 at 15 03 55

Thanks for merging!

**Pull request merged**